ts/components/search/SearchButton.tsx

```diff
@@ -25,6 +25,7 @@ export type SearchType = "Messages" | "Services";
 interface OwnProps {
   color?: string;
   searchType?: SearchType;
+  onSearchTap?: () => void;
```

Comment:
```suggestion
    // if this handler is defined it will be called in place of dispatching actions about search activation (see handleSearchPress)
    onSearchTap?: () => void;
```

Reply:
We discussed about it but I forgot to add! Thanks!